client/src/utils/constants.js

```diff
@@ -238,4 +238,33 @@ export const states = [
   },
 ];
 
+export const dataSet = [
+  {
+    columns: [
+      'Date',
+      'Daily Feeling',
+      'Daily Symptoms Feeling',
+      'Daily Compared to Yesterday',
+      'Fever Severity',
+      'Shortness of Breath Severity',
+      'Chills Severity',
+      'Cough Severity',
+      'Fatigue Severity',
+      'Sore Throat Severity',
+      'Bluishness Severity',
+      'GI Issues Severity',
```

Comment:
```suggestion
      'Gastrointestinnal Issues Severity',
```

If you think it needs to be more readable